capstone/src/main/java/com/google/univiz/servlets/SearchService.java

```diff
@@ -38,4 +40,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     List<SearchData> searchResults = searchResource.getSearchSuggestions(query);
     ServletHelper.writeJsonToResponse(gson, response, searchResults);
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String numberCollegesStr = request.getParameter("collegeNumber");
+    int numberOfColleges = Integer.parseInt(numberCollegesStr);
+    List<String> listToSend = new ArrayList<>();
+    for (int i = 0; i < numberOfColleges; i++) {
+      StringBuilder parameterStr = new StringBuilder();
+      parameterStr.append("college");
+      parameterStr.append(Integer.toString(i));
```

Comment:
Should there be a separator, like a colon or comma?

Reply:
for the parameter name? no-I made the parameters register themselves as 'college0', 'college1', etc.

Reply:
Oh I see, that makes sense then